File: C/Code/2500/2562 - 최댓값.c
#include <stdio.h>

int main() {
	int max = 0, maxnum;
	for (int i = 0; i < 9; i++) {
		int temp;
		scanf("%d", &temp);				// input
		if (arr[i] > max)				// if max value
			max = temp, maxnum = i;		// change result
	}
	printf("%d\n%d", max, maxnum + 1);
	return 0;
}

File: Python/Code/2500/2562 - 최댓값.py
a = []
for i in range(9):
	a.append(int(input()))
b = a[0]
c = 1
for i in range(len(a)):
	if a[i] > b:
		b = a[i]
		c = i + 1
print("%d\n%d" % (b, c))